tests/snapshots/releasenotes/notes/asm-flask-ip-blocking-a2416d04cacf71f3.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I think this file might be in the wrong location. It probably belongs in `/releasenotes`.